app/src/main/res/navigation/nav_graph.xml

```diff
@@ -133,6 +133,12 @@
             app:exitAnim="@anim/nav_default_exit_anim"
             app:popEnterAnim="@anim/nav_default_pop_enter_anim"
             app:popExitAnim="@anim/nav_default_pop_exit_anim" />
+
+        <argument
```

Comment:
the value needs to be somewhere declared as true to show it in fullscreen

Reply:
it was already used before, I had just accidentally removed the argument.
It is used in
private fun openHelpPage() {
        navigate(
            R.id.action_nav_welcome_fragment_to_nav_help,
            Bundle().apply { putBoolean("fullscreen", true) })
    }
In welcome fragment.
Tested in Help and also when deactivated -> Works OK.

Reply:
But the problem is not only in "Help" but in "About Application" where you can get directly from "Help".

Reply:
_HelpVM.kt_
```
is AboutAppCategory -> {
                navigate(
                    R.id.nav_about,
                    Bundle().apply { putBoolean("fullscreen", isFullscreen) }
                )
            }
```

I suggest smth like that:
_HelpFragment.kt_
```
override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        viewModel.fillInHelp()

        viewModel.isFullscreen = arguments?.let {
            HelpFragmentArgs.fromBundle(it).fullscreen
        } ?: false

    }
```